include/mbedtls/md.h

```diff
@@ -99,7 +105,7 @@ typedef struct mbedtls_md_context_t
     const mbedtls_md_info_t *md_info;
 
     /** The digest-specific context. */
-    void *md_ctx;
+    void *MBEDTLS_PRIVATE(md_ctx);
```

Comment:
This is likely to confuse source code navigation. I don't know if this really matters: do we often look up field names (as opposed to function or type names) when working on the library?

Reply:
I can't speak for others, but I never look up field names. So I don't think the disruption to source code navigation is a big issue here.